Makefile

```diff
@@ -9,14 +19,20 @@ aws-deps:
 azure-deps: jq-dep
 	npm install -g azure-cli
 
-jq-dep:
+jq-dep: $(GOPATH)/bin/jq
+
+$(GOPATH)/bin/jq:
 	@echo "Downloading jq..."
 	mkdir -p $(GOPATH)/bin
 	wget "https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64" -O $(GOPATH)/bin/jq
 	chmod "+x" $(GOPATH)/bin/jq
```

Comment:
Is it a real feeling ot not? hahaha

I put there because of I'm trying to use the `travis ci container infra` to run tests, and it disallow root access. GOPATH is already there because of `lang: go` in the .travis.yml file...

Alternative location?